visidata/loaders/_pandas.py

```diff
@@ -61,3 +132,19 @@ def open_dta(p):
     return PandasSheet(p.name, source=p, filetype='stata')
 
 open_stata = open_pandas
+
+undoSheetSelection = undoAttrCopy('[sheet]', '_selectedMask')
+
+# Override with vectorized implementations
+PandasSheet.addCommand('gt', 'stoggle-rows', 'toggleByIndex()', undo=undoSheetSelection),
```

Comment:
I *think* these command redefinitions should be `None` for the keybinding, so that if someone binds `gt` globally to another longname it won't get rebound here on the PandasSheet.  But we'd have to make sure that `gt` etc still properly bind to the longname on the PandasSheet, of course (and I'll have to fix the keybinding code if they don't).

Reply:
Done, I haven't looked at the keybinding code but setting to `None` seems to work fine for what I've tried.